Esterni/PianoProgetto/components/Introduzione.tex

```diff
@@ -0,0 +1,39 @@
+\section{Introduzione}
+\subsection{Scopo del documento}
+Lo scopo del documento è quello di specificare la pianificazione che \Gruppo  adotterà per sviluppare il progetto \NomeProgetto. All'interno sono presenti le strategie per la gestione dei rischi, le suddivisioni dei compiti con il relativo preventivo dell'utilizzo delle risorse, e il modello di sviluppo secondo il quale il team ha intenzione di lavorare.
```

Comment:
Come già detto il nome progetto deve essere completo ed in corsivo (quindi da cambiare la macro di \NomeProgetto

Reply:
risolto